README.md

```diff
@@ -1,49 +1,73 @@
 # Meetbot 
 
-Meetbot will listen to google meet urls to join to and perform various tasks.
+Meetbot is a Google Meet bot that makes meetings frictionless. 
 
-By default since the bot isn't authenticated it will prompt people in the meet to allow the bot to join.
+Meetbot aims to eliminate questions in video calls like:
+
+_1. Can you hear me? Can you hear us?_ Validates audio input of attendees minus the awkward silence.
+_2. Can you resend those links again?_ Meetbot records chat transcript.
+_3. Do you have any notes from the call?_ Also, records voice caption transcript.
+_4. Was the meeting recorded?_ Auto-records meetings. 
+_5. Does anyone have the links shared in the meeting?_ Saves it all to Google Docs. Easy sharing.
+
+And, many more features.
 
 ![](img/diagram.drawio.png)
 
-## Development
+## Try it out!
 
-After cloning the repository, install the dependencies:
+Running this project is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
```

Comment:
That makes it sound like you wouldn't need a device to deploy to

Reply:
Copied it from a balenaLabs project, but you are indeed correct.

Reply:
I guess with most labs projects it's kind of obvious from the start that it focuses running on a device, whereas MeetBot could just as well be another cloud service